velox/substrait/tests/Substrait2VeloxValuesNodeConversionTest.cpp

```diff
@@ -34,33 +31,22 @@ using namespace facebook::velox::substrait;
 
 class Substrait2VeloxValuesNodeConversionTest : public OperatorTestBase {
  public:
-  void parseJson(const std::string& filePath, ::substrait::Plan* subPlan) {
```

Comment:
Nice refactor. It would be helpful to submit this as a separate PR. This will make it easier to review.

Reply:
Removed this change. Will submit this in next PR.

Reply:
Raised this change to a new PR: https://github.com/facebookincubator/velox/pull/1637.